io/disk/scsi_add_remove.py

```diff
@@ -33,44 +36,62 @@ def setUp(self):
         '''
         Function for preliminary set-up to execute the test
         '''
+        self.err_paths = []
         smm = SoftwareManager()
         if not smm.check_installed("lsscsi") and not smm.install("lsscsi"):
             self.cancel("lsscsi is not installed")
-        self.pci_device = self.params.get("pci_device", default=None)
-        if not self.pci_device:
-            self.cancel("Please provide PCI address for which you \
-                        want to run the test")
+        self.wwids = self.params.get('wwids', default='').split(',')
+        system_wwids = multipath.get_multipath_wwids()
+        for wwid in self.wwids:
+            if wwid not in system_wwids:
+                self.cancel("%s not present in the system" % wwid)
```

Comment:
I am still not fine with the assumption that it is only multipath devices that the test considers.

```
# multipath -ll
Jul 10 12:13:59 | DM multipath kernel driver not loaded

# lsscsi -u
[0:0:0:0]    disk    5000c500a8f849f9                  /dev/sda 
[1:0:0:0]    disk    5000c500a8f61901                  /dev/sdb 
[N:0:33:1]   disk    335256304a6002060025384100000001           /dev/nvme0n1
[N:0:33:26]  disk    335256304a600206002538410000001a           /dev/nvme0n2
```